udaykalra/portfolio/src/main/java/com/google/sps/servlets/PollDataServlet.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/endorse-data")
+public class PollDataServlet extends HttpServlet {
```

Comment:
Style requires documentation for public classes:
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Documentation would also be useful to the reader to understand that "Poll" means "Endorsed Skills".

Optional: renaming this to EndorsedSkillsDataServlet may be clearer.

Reply:
Done